CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+### Fixed
+- Changed the last uses of 4.4.2 in setup.py and schema.yaml. ([#4172](https://github.com/wazuh/wazuh-qa/pull/4172)) \- (Framework + Tests)
```

Comment:
```suggestion
- Changed the last uses of 4.4.2 in setup.py and schema.yaml. ([#4172](https://github.com/wazuh/wazuh-qa/pull/4172)) \- (Framework)
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4172/commits/2bcf993831339313d2da62c697177ef1fa0ad3da